src/cards/community/corporations/TempestInc.ts

```diff
@@ -127,6 +128,27 @@ export class TempestInc extends Card implements CorporationCard {
       }));
     }
 
+
+    if (player.game.gameOptions.moonExpansion === true) {
+      orOptions.options.push(new SelectOption('Increase Moon Colony rate', 'Select', () => {
+        player.removeResourceFrom(this, 1);
+        MoonExpansion.raiseColonyRate(player);
```

Comment:
Note that raiseColonyRate takes care of logging. I suppose I should take care of not showing the option if it's already at max.

Reply:
Ah never mind about "not showing the option if it's already at max"